src/planet-a/icosahedron.h

```diff
@@ -91,31 +91,44 @@ inline constexpr std::array<std::array<uint8_t, 3>, 20> gc_icoNeighbors
 /**
  * @brief Icosahedron Minimum Edge Length vs Subdiv Levels, radius = 1.0
  */
-inline constexpr std::array<float, 10> const gc_icoMinEdgeVsLevel
+inline constexpr std::array<float, 24> const gc_icoMinEdgeVsLevel
 {
-    1.05146222e+0f,  5.46533058e-1f,  2.75904484e-1f,  1.38283174e-1f,  6.91829904e-2f,
-    3.45966718e-2f,  1.72989830e-2f,  8.64957239e-3f,  4.32479631e-3f,  2.16239942e-3f
+    1.05146222e+0f,  5.46533058e-1f,  2.75904484e-1f,  1.38283174e-1f,
```

Comment:
could you document where the initial values of this are from? Is it `icosahedron_tables.py` ?

Reply:
There's already comment; it's just not visible on the diff.